src/profile/v100/identified_elements.rs

```diff
@@ -354,6 +354,18 @@ macro_rules! decl_can_be_empty_impl {
     ) => {
         paste! {
 
+            impl FromIterator<$element_type> for $struct_type {
```

Comment:
Should ofc have conformed to `FromIterator` protocol, it is a great trait, allowing us to do `.collect`!!